maintainers/scripts/copy-tarballs.pl

```diff
@@ -20,14 +20,46 @@
 use File::Slurp;
 use JSON;
 use Net::Amazon::S3;
-use Nix::Store;
-
-isValidPath("/nix/store/aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa-foo"); # FIXME: forces Nix::Store initialisation
 
 sub usage {
     die "Syntax: $0 [--dry-run] [--exclude REGEXP] [--expr EXPR | --file FILES...]\n";
 }
 
+sub isValidPath {
+    my ($path) = @_;
+    my $isValid = system("nix-store", "--verify-path", $path) == 0;
```

Comment:
The original call did not check the contents, and that seems to be ok.
I don't think we have an exact command line equivalent except `nix path-info`, but that's experimental.

```suggestion
    my $isValid = system("nix-store", "--size", $path) == 0;
```

Reply:
We can actually always just call nix-store -r.